flow/platforms/ihp-sg13g2/sg13g2-update.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import os
+
+def download_github_file(repo_owner, repo_name, branch, file_path, local_dir):
```

Comment:
Make arguments optional by adding default values. Simplifies `download_github_file` calls a lot.


```suggestion
def download_github_file(repo_owner, repo_name, branch, file_path, local_dir):
```
```suggestion
REPO_OWNER= 'IHP-GmbH'
REPO_NAME= 'IHP-Open-PDK'
REPO_BRANCH = 'dev'

def download_github_file(file_path, local_dir='.', repo_owner=REPO_OWNER, repo_name=REPO_NAME, branch=REPO_BRANCH):
```